media/com_finder/js/maps.es6.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+Joomla = window.Joomla || {};
+
+(() => {
+  'use strict';
+
+  document.addEventListener('DOMContentLoaded', () => {
+    Joomla.submitbutton = (pressbutton) => {
+      if (pressbutton === 'map.delete') {
+        if (window.confirm(Joomla.JText._('COM_FINDER_MAPS_CONFIRM_DELETE_PROMPT'))) {
+          Joomla.submitform(pressbutton);
+        } else {
+          return false;
+        }
+      }
+      Joomla.submitform(pressbutton);
+      return true;
```

Comment:
Same as in https://github.com/joomla-projects/joomla-es6/pull/90#pullrequestreview-102567078